meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: tomopy
-  version: '0.1.8'
+  version: '0.1.9'
 
 source:
   git_url: https://github.com/dgursoy/tomopy.git
```

Comment:
Thanks Tom for the detailed information. i have few more points to ask but we can discuss more on this in few days.